Mathlib/Data/List/PeriodicityLemma.lean

```diff
@@ -0,0 +1,236 @@
+/-
+Copyright (c) 2025 Štěpán Holub. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Štěpán Holub
+-/
+module
+
+public import Mathlib.Algebra.Order.Group.Nat
+public import Mathlib.Data.Nat.Lattice
```

Comment:
```suggestion
public import Mathlib.Data.Nat.Lattice
public import Mathlib.Tactic.TacticAnalysis.Declarations
```

We are in the process of swapping out `omega` in favour of `lia`.
This import will be needed during the transition phase.

In the remainder of the file, every call to `omega` can be replaced by `lia`, except for the last one, which is can be replaced by `grind`.